.github/workflows/release.yml

```diff
@@ -15,8 +15,8 @@ jobs:
         with:
           fetch-depth: 0
 
-      - name: Use Xcode 14.3.1
-        run: sudo xcode-select -switch /Applications/Xcode_14.3.1.app
+      - name: Use Xcode 15
+        run: sudo xcode-select -switch /Applications/Xcode_15.0.app
```

Comment:
This shouldn't cause any breaking changes since we do not build any binaries/support Carthage for DropIn